src/nixpacks/plan/generator.rs

```diff
@@ -18,6 +18,7 @@ static NIXPKGS_ARCHIVE: &str = "41cc1d5d9584103be4108c1815c350e07c807036";
 
 #[derive(Clone, Default, Debug)]
 pub struct GeneratePlanOptions {
+    pub custom_install_cmd: Option<String>,
```

Comment:
This isn't provided anywhere. It should also probably be an option in the CLI, similar to `--build-cmd`